src/extension/background-script/WelcomeService.ts

```diff
@@ -228,3 +228,8 @@ export async function openWelcomePage(id?: SocialNetworkUI['lastRecognizedIdenti
     }
     return browser.tabs.create({ url: getWelcomePageURL(id) })
 }
+
+export async function openOptionsPage(url: string) {
```

Comment:
openExtensionPage?

Reply:
they are synonyms since we don't expect user to open background page or popup page

Reply:
`(url: string)`

Reply:
maybe `/index.html#/welcome` or something. ok I'll change that